scripts/check-techniques.sh

```diff
@@ -146,3 +146,11 @@ do
stribution/checkGenericFileContent/4.0 fileDistribution/checkGenericFileContent/5.0 fileDistribution/checkGenericFileContent/6.0 fileDistribution/copyGitFile/1.0 fileDistribution/copyGitFile/1.1 fileDistribution/copyGitFile/1.2 fileDistribution/copyGitFile/1.3 fileDistribution/copyGitFile/1.4 fileDistribution/copyGitFile/1.5 fileDistribution/copyGitFile/1.6 fileDistribution/copyGitFile/1.7 fileDistribution/downloadFile/1.0 fileDistribution/downloadFile/2.0 systemSettings/misc/clockConfiguration/1.0 systemSettings/misc/clockConfiguration/2.0 systemSettings/misc/genericCommandVariableDefinition/1.0 systemSettings/misc/genericCommandVariableDefinition/2.0 systemSettings/misc/genericVariableDefinition/1.0 systemSettings/misc/genericVariableDefinition/1.1 systemSettings/misc/genericVariableDefinition/1.2 systemSettings/misc/partitionSizeMonitoring/1.0 systemSettings/misc/partitionSizeMonitoring/2.0 systemSettings/networking/dnsConfiguration/1.0 systemSettings/networking/dnsConfiguration/1.1 systemSettings/networking/dnsConfiguration/2.0 systemSettings/networking/hostsConfiguration/1.0 systemSettings/networking/hostsConfiguration/1.1 systemSettings/networking/nfsClient/1.0 systemSettings/networking/nfsClient/2.0 systemSettings/networking/nfsServer/1.0 systemSettings/networking/nfsServer/2.0 systemSettings/networking/routingManagement/1.0 systemSettings/process/processManagement/1.0 systemSettings/process/processManagement/1.1 systemSettings/process/processManagement/2.0 systemSettings/process/servicesManagement/1.0 systemSettings/process/servicesManagement/1.1 systemSettings/process/servicesManagement/1.2 systemSettings/process/servicesManagement/2.0 systemSettings/remoteAccess/sshConfiguration/1.0 systemSettings/remoteAccess/sshConfiguration/2.0 systemSettings/remoteAccess/sshConfiguration/3.0 systemSettings/remoteAccess/sshKeyDistribution/1.0 systemSettings/remoteAccess/sshKeyDistribution/2.0 systemSettings/security/fileAlterationMonitoring/1.0 systemSettings/systemManagement/cronManagement/1.0 systemSettings/systemManagement/cronManagement/2.0 systemSettings/systemManagement/fstabConfiguration/1.0 systemSettings/systemManagement/fstabConfiguration/2.0 systemSettings/systemManagement/fstabConfiguration/3.0 systemSettings/systemManagement/motdConfiguration/1.0 systemSettings/systemManagement/motdConfiguration/2.0 systemSettings/systemManagement/motdConfiguration/3.0 systemSettings/userManagement/groupManagement/1.0 systemSettings/userManagement/groupManagement/2.0 systemSettings/userManagement/groupManagement/3.0 systemSettings/userManagement/groupManagement/3.1 systemSettings/userManagement/groupManagement/4.0 systemSettings/userManagement/sudoParameters/1.0 systemSettings/userManagement/sudoParameters/2.0 systemSettings/userManagement/userManagement/1.0 systemSettings/userManagement/userManagement/2.0 systemSettings/userManagement/userManagement/3.0 systemSettings/userManagement/userManagement/4.0 systemSettings/userManagement/userManagement/5.0; do
```

Comment:
The process would be :
* we add a presence check in 2.11
* the check is upmerged
* we remove a technique in 3.0
* -> the presence test fails
* we know we need to change the test so we change the presence test into absence test
* a technique is wrongly upmerge
* -> the test alerts us